build.sbt

```diff
@@ -21,7 +21,7 @@ lazy val commonSettings = Seq(
   organization := "com.cognite",
   organizationName := "Cognite",
   organizationHomepage := Some(url("https://cognite.com")),
-  version := "2.0.10",
+  version := "2.0.11",
```

Comment:
Bump to `2.1.0`, we'll get compilation errors on projects using those classes when Scala steward does automatic updates.